other/setup/ubuntu.sh

```diff
@@ -1,8 +1,11 @@
 #!/bin/bash
+
+set -e
+
 export GIT_SSL_NO_VERIFY=1
 
 sudo apt-get update
-sudo apt-get -y install build-essential linux-headers-$(uname -r) wget tar libboost-dev cmake 
```

Comment:
`uname -r` does not give the result you expect when invoked in a container. For example, with Docker and Ubuntu 18.04: `4.18.9-arch1-1-ARCH`. I changed it to `generic`, which has the additional benefit of not becoming outdated when the kernel is upgraded (also in non-containerized setups). You may want to check whether this works for you.